package.json

```diff
@@ -64,6 +64,11 @@
   "activationEvents": [
     "onCommand:pacCLI.openDocumentation",
     "onCommand:microsoft-powerapps-portals.webExtension.init",
+    "onCommand:powerplatform-walkthrough.overview-learn-more",
```

Comment:
It looks like you're referencing these commands here, and registering the handlers in the extension.ts, but never actually declaring the commands in the `contributes -> commands` section of the package.json

Reply:
Yes these are kind of internal commands and are invoked by clicking on links/buttons described in the package.nls.json file.
E.g. [Learn More](command:powerplatform-walkthrough.overview-learn-more)

Please check the official VSCode samples: https://github.com/microsoft/vscode-extension-samples/blob/main/getting-started-sample/package.json

Reply:
Interesting.   I assume that all commands needed to be listed in the Contributes section, but it looks like that's only necessary when the command needs a title or icon, as all of the commands we've previously made have but that these do not.

Double checked by running it locally, and it all appears to work fine.